### PR TITLE
fix bug with components not rerendering

### PR DESCRIPTION
**Pull request by zachjamesgreen**
'fix' bug where cocktail component would not update/rerender 

- [x] Build is passing

## Pull request type
Please check the type of change your PR introduces:
- [x] Bugfix
## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/components/cocktail.js [hunk 1]
```diff
@@ -6,22 +6,31 @@ import { GET_A_RECOMMENDATION } from '../constants.js';
 import AppDispatcher from '../core/dispatcher.js';
 import cocktailStore from '../stores/cocktail_store.js';
 import spinnerStore from '../stores/spinner_store.js';
+import { Link } from 'react-router-dom'
 
 export default class Cocktail extends React.Component {
 
   constructor(props) {
     super(props);
+    console.log(props);
```
Comment (mkrumholz) [inline src/components/cocktail.js:15]: Don't forget to remove the logs!

File: src/components/cocktail.js [hunk 2]
```diff
@@ -6,22 +6,31 @@ import { GET_A_RECOMMENDATION } from '../constants.js';
...
   updateQ = (e) => {
     this.setState({loc: this.loc.current.value})
   }
 
+  // shouldComponentUpdate = (nextProps, nextState) => {
+  //   console.log(this.props, this.state);
+  //   console.log(nextProps, nextState);
+  //   return true;
+  // }
+
```
Comment (mkrumholz) [inline src/components/cocktail.js:33]: Is this for a future iteration or something we'll use later?

Comment (zachjamesgreen) [reply src/components/cocktail.js:33]: it was the old code that may not be needed anymore but I didn't want to delete it out of the file yet. I guess I can

Comment (zachjamesgreen) [reply src/components/cocktail.js]: removed

File: src/components/cocktail.js (hunk 1 above)
Comment (zachjamesgreen) [reply src/components/cocktail.js]: removed

**Review by mkrumholz — APPROVED** [review]
Thanks! Looks good to me, though I'm having trouble testing on local, so the second reviewer might want to do that.

**Review by twarbelow — APPROVED** [review]

**Pull request merged**